packages/lb-components/src/components/NLPToolView/index.tsx

```diff
@@ -114,6 +114,16 @@ const NLPToolView: React.FC<IProps> = (props) => {
     updateSidebar()
   }
 
+  const clearResult = () => {
```

Comment:
Why is it not an empty list?

Reply:
clearResult means textAnnotation is an empty list